src/operator/src/statement/ddl.rs

```diff
@@ -702,6 +702,7 @@ impl StatementExecutor {
         catalog: &str,
         database: &str,
         create_if_not_exists: bool,
+        options: Option<HashMap<String, String>>,
```

Comment:
`HashMap` would be enough:

```suggestion
        options: HashMap<String, String>,
```